res/commands.json

```diff
@@ -89,9 +89,9 @@
 		"additional": "",
 		"admin:": false
 	},
-	"USER_MODULE_STATS": {
+	"USER_MODULE_RECORDS": {
 		"brief": "Shows statistics about a user's time in the server.",
-		"usage": "stats <Person>",
+		"usage": "records <Person>",
```

Comment:
Why did you rename it to "records"?

Reply:
It is in the TODO, to change that. I makes confusion with the `!status` command. 🙂